main.cpp

```diff
@@ -73,12 +23,23 @@ void testPairToNumber(
     assert(pairNumber == expectedPairNumber);
 }
 
+void ToString()
+{
+    int i;
+    std::cout << std::endl << "----------MANUAL----------" << std::endl;
+    for(i=1;i<=25;i++)
```

Comment:
Do not hard-code the loop limit. it will be hard to remember updating this every time the colors change.